cogs/DeepL.py

```diff
@@ -1,63 +1,69 @@
-from deepl import translate
+from typing import Optional
```

Comment:
`def x(c: str or None = None)` is preferred over importing the typing module for one optional typehint